konduit-serving-api/src/main/java/ai/konduit/serving/pipeline/PythonPipelineStep.java

```diff
@@ -93,8 +93,8 @@ public PythonPipelineStep(PythonConfig pythonConfig, String[] inputColumnNames,
     public PythonPipelineStep(Schema inputSchema, Schema outputSchema,
                               PythonConfig pythonConfig) throws Exception {
         String defaultName = "default";
-        this.setInput(defaultName, inputSchema);
-        this.setOutput(defaultName, outputSchema);
```

Comment:
same here, you decide. we can leave it like this.

Reply:
c9232f6